lib/modules/versioning/hermit/readme.md

```diff
@@ -0,0 +1,13 @@
+Hermit versioning is a mix of `semver` and `channel`.
+
+**Semver**
+
+Hermit's semver was the untangled versions comes back from package's tag.
```

Comment:
I don't think this sentence makes sense.

Reply:
I've reworded the section. PTAL.